cedar-drt/tests/benchmark.rs

```diff
@@ -0,0 +1,182 @@
+                    auth_results.push(response.timing_info["authorize"] as f64);
+                }
+            }
+            results
+                .get_mut("total")
+                .unwrap()
+                .push(median(total_results));
+            results
+                .get_mut("authorize")
+                .unwrap()
+                .push(median(auth_results));
+        }
+    }
+    results
+}
+
+/// Run every input in the corpus tests through the provided Cedar test implementation,
+/// recording the time(s) required for validation over NUM_TRIALS trials. The
+/// output reports the median trial value for each input.
+///
+/// There are two statistics measured: "total" for end-to-end validation time
+/// and (optionally) "validate" for the reported _core_ validation time, which
+/// ignores time required to interface with the test implementation.
+fn get_validation_timing_results(
+    custom_impl: &dyn CedarTestImplementation,
+) -> HashMap<&str, Vec<f64>> {
+    let tests = get_corpus_tests();
+    let mut results = HashMap::new();
+    results.insert("total", Vec::new());
+    results.insert("validate", Vec::new());
+    for test in tests {
+        let (policies, _entities, schema, _requests) = parse_test(test);
+        let mut total_results = Vec::new();
+        let mut val_results = Vec::new();
+        for _i in 0..NUM_TRIALS {
+            let (response, duration) =
+                time_function(|| custom_impl.validate(&schema, &policies, ValidationMode::Strict));
+            let response = response.expect("Unexpected test failure");
+            total_results.push(duration.as_micros() as f64);
+            if response.timing_info.contains_key("validate") {
+                val_results.push(response.timing_info["validate"] as f64);
+            }
+        }
+        results
+            .get_mut("total")
+            .unwrap()
+            .push(median(total_results));
+        results
+            .get_mut("validate")
+            .unwrap()
+            .push(median(val_results));
+    }
+    results
+}
+
+/// Print out a summary of the results
+fn print_summary(auth_times: HashMap<&str, Vec<f64>>, val_times: HashMap<&str, Vec<f64>>) {
+    for (key, value) in auth_times.into_iter() {
+        println!("Authorization - {key}");
+        println!("\tMedian: {:.1} micros", median(value.clone()));
+        println!("\tp90: {:.1} micros", p90(value.clone()));
+        println!("\tp99: {:.1} micros", p99(value));
+    }
+    for (key, value) in val_times.into_iter() {
+        println!("Validation - {key}");
+        println!("\tMedian: {:.1} micros", median(value.clone()));
+        println!("\tp90: {:.1} micros", p90(value.clone()));
+        println!("\tp99: {:.1} micros", p99(value));
+    }
+}
+
+#[test]
+// Currently, running this in conjunction with existing tests will cause an error (#227).
+// In order see the printed output from this test, run `cargo test -- --ignored --nocapture`.
+// #[ignore]
```

Comment:
you don't need `--ignored` if `#[ignore]` is commented out like this.  we should either un-comment-out the `#[ignore]` or remove the `--ignored`

Reply:
Aha that explains the build failure. I guess I commented it out to double check my claim in #227.